apps/condo/next.config.js

```diff
@@ -106,6 +107,7 @@ let nextConfig = withTM(withLess(withCSS({
         termsOfUseUrl,
         privacyPolicyUrl,
         dataProcessingConsentUrl,
+        isSnowfallDisabled,
```

Comment:
Can we turn it on in runtime without restarts?

Reply:
You can do this by setting the number of snowflakes to zero, via the feature flag (default is zero).

A flag from environment variables allows you to completely disable the snowfall logic from the application